docs/lifecycle-events.md

```diff
@@ -1,59 +1,73 @@
 # Lifecycle Events
 
-Lifecycle events are custom function handlers invoked at various points in the life of a [virtual node](/docs/virtual-nodes.md).
-
-<pre>
-<i>event</i>(<a href="https://developer.mozilla.org/en-US/docs/Web/API/Element">Element</a>)
-</pre>
+Lifecycle events are custom function handlers invoked at various points in the life of a [virtual node](/docs/virtual-nodes.md). They are used like any other [attribute](/docs/api.md#attributes).
 
 ## oncreate
 
-The oncreate event is fired when the element is created, but before it is inserted into the DOM. Use this event to start animations before an element is rendered.
-
-## oninsert
-
-The oninsert event is fired after the element is created and inserted into the DOM. Use this event to wrap third party libraries that require a reference to a DOM node, etc.
+The [oncreate](/docs/api.md#oncreate) event is fired after the element is created, but before it is attached to the DOM. Use it to start animations before an element is rendered, [adapt an external library](#adapting-an-external-library) that requires a reference to a DOM node, etc.
```

Comment:
We're changing this right? It's going to be called *after* element has been attached to the dom, right?